### ISSUE TITLE
Display avatars of W3C staff too?

### ISSUE THREAD

**Issue by tripu** [w3c/mailing-list-archives#22]
Similar to #21, but specifically for the case of people with `@w3.org` addresses.

See how we can pull the portrait of the sender from [`w3.org/People`](https://www.w3.org/People/) and display it on the page.

Question: do we _want_ to do that?
(My personal opinion: yes; it is useful; it lets the user recognise the sender at a glance, without even reading the &ldquo;from&rdquo; field; it contributes to a better corporate image and to identify the W3C team; and all those images are public now anyway.)

**Comment by gbaudusseau** [comment]
+1. It could be fun if we could use the W3C API for that :)

**Comment by tripu** [comment]
Yes, we could use the API for that; user entities have a property `photos`.
eg, [`https://api.w3.org/users/ggpn34zlppss0g4480g0skc4k4oo4o8?_format=json`](https://api.w3.org/users/ggpn34zlppss0g4480g0skc4k4oo4o8?_format=json) (append your API key).
We'd have to translate staff e-mail address into user hash, though.

**Comment by tguild** [comment]
I like a mix of Gravatar and W3C account profile photo accessed through our Web API if implementing both are not too onerous.  Which should be given preference though?  I would think the W3C one as it is explicitly the image someone wants to represent them at W3C.

**Comment by tripu** [comment]
(Yes, I'd say our W3C profile image should take precedence over Gravatar too.)

**Comment by ZoeBijl** [comment]
+1 to W3C profile image; I don't a gravatar.
